src/schema/partner_show.ts

```diff
@@ -93,9 +104,49 @@ const PartnerShowType = new GraphQLObjectType({
           ).then(({ body }) => body)
         }
 
+        // @ts-ignore
+        // FIXME: Update with Gravity param when ready
         return fetch.then(exclude(options.exclude, "id"))
       },
     },
+    artworksConnection: {
+      description: "A connection of artworks featured in the show",
+      type: artworkConnection,
+      args: pageable(artworksArgs),
```

Comment:
Theoretically, `page`, `size` (included in `artworksArgs`) are not valid arguments to a connection.

Reply:
The special `all` arg in that hash also doesn't apply here.

Reply:
Fixed this by merging whats needed into the `artworks` has above